packages/video_player/lib/video_player.dart

```diff
@@ -136,20 +136,21 @@ class VideoPlayerController extends ValueNotifier<VideoPlayerValue> {
       );
     }
 
-    void eventListener(Map<String, dynamic> event) {
-      if (event["event"] == "initialized") {
+    void eventListener(dynamic event) {
+      final Map<String, dynamic> message = event;
```

Comment:
Again, the Map doesn't really represent a message. Maybe refactoring to a function can make this more explicit, e.g.
```dart
VideoPlayerValue toUpdatedValue(dynamic update, VideoPlayerValue currentValue) { ... }
```

Reply:
I renamed message=>map. I am not sure that all events should be expressible on that form even if that's the case right now.